core/EE_System.core.php

```diff
@@ -854,11 +854,6 @@ public function load_core_configuration()
         // load and setup EE_Config and EE_Network_Config
         $config = $this->loader->getShared('EE_Config');
         $this->loader->getShared('EE_Network_Config');
-        // setup autoloaders
-        // enable logging?
-        if ($config->admin->use_full_logging) {
```

Comment:
hmmmm... we should likely change this to:

```php
if ($config->admin->use_remote_logging) {
    $this->loader->getShared('EE_Log');
}
```

instead of removing it because otherwise that won't work at all

Reply:
Nice catch!